### PR TITLE
Give an example of limiting the CPU of the cpu_hog bad actor

### PR DESCRIPTION
**Pull request by emersonknapp**
Add a new launchfile that shows how to apply a CPU constraint on a sandboxed node. Update documentation and byobu config to show how to use it.

### CODE REVIEW

File: configs/sandbox_demo/bad_actor.conf [hunk 1]
```diff
@@ -9,4 +9,7 @@ setenv FOO "foo"
 source configs/sandbox_demo/security_workshop_demo_common.conf
 
 new-window -n bad_actor
-send-keys 'ros2 run example_nodes cpu_hog 8'
+send-keys 'ros2 run example_nodes cpu_hog 32'
+
+split-window -v
+send-keys 'ros2 launch example_nodes sandboxed_cpu_hog.launch.py'
```
Comment (dabonnie) [inline configs/sandbox_demo/bad_actor.conf:15]: We could spawn a new window and run top / glances automatically.

**Review by thomas-moulard — APPROVED** [review]

File: configs/sandbox_demo/bad_actor.conf (hunk 1 above)
Comment (thomas-moulard) [reply configs/sandbox_demo/bad_actor.conf:15]: That'd be interesting...

Comment (emersonknapp) [reply configs/sandbox_demo/bad_actor.conf:15]: This is an extension of the main configuration, so the glances window titled "diagnostics" is running.

Comment (dabonnie) [reply configs/sandbox_demo/bad_actor.conf:15]: Do we need this in the readme then?

>If you run the top command (`ros2 run...`), you can see the CPU of your system jump to 100% usage, grinding everything else to a halt.

Just wondering if the info is already present in glances.

File: example_nodes/launch/sandboxed_cpu_hog.launch.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                    node_executable='cpu_hog',
+                ),
+            ]
+        )
+    )
+
+    return ld
+
+
+if __name__ == '__main__':
```
Comment (piraka9011) [inline example_nodes/launch/sandboxed_cpu_hog.launch.py:67]: Is this supposed to be run as a launch file or standalone executable?

I see it being used as a launch file so I don't think this is needed.

Comment (emersonknapp) [reply example_nodes/launch/sandboxed_cpu_hog.launch.py:67]: the main entrypoint is for debug purposes only - launch is just candy around this exact code so it's usable either way - you notice in there that it's got debug flags.

intended use case is via `ros2 launch` but sandboxing still has enough potential bugs that it's useful to quickly have this debug entrypoint

**Review by piraka9011 — APPROVED** [review]

File: configs/sandbox_demo/bad_actor.conf (hunk 1 above)
Comment (emersonknapp) [reply configs/sandbox_demo/bad_actor.conf:15]: Yeah the info is already there - and it's called out higher up in README in the intro to the overall demo. I've just pushed a commit with a little hint to remember that it exists.

**Pull request merged**